scripts/rook.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash -E
 
-ROOK_VERSION=${ROOK_VERSION:-"v1.2.7"}
+ROOK_VERSION=${ROOK_VERSION:-"v1.4.9"}
 ROOK_DEPLOY_TIMEOUT=${ROOK_DEPLOY_TIMEOUT:-300}
 ROOK_URL="https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/cluster/examples/kubernetes/ceph"
 ROOK_BLOCK_POOL_NAME=${ROOK_BLOCK_POOL_NAME:-"newrbdpool"}
-ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
+ROOK_CEPH_CLUSTER_VERSION=${ROOK_CEPH_CLUSTER_VERSION:-"v14.2.12"}
```

Comment:
as @nixpanic suggested If `ROOK_CEPH_CLUSTER_VERSION` is not set extract the ceph version from `BASE_IMAGE`  and use it.

Reply:
ok